### DISCUSSION TITLE
Issue when translating a top category

### DISCUSSION THREAD

**Discussion by Nek-** [Sylius/Sylius#13021] [Q&A]
Hello,

I switched the default locale (parameter %locale%) to a new language but many parts of the app are now blank and need to be filled, this is the case for taxons:
![image](https://user-images.githubusercontent.com/972456/131139599-ccb529a0-c05a-4aec-882e-003198bbfcd3.png)

The problem here is that when trying to translate the first taxon the slug generation end up in 500 error. After digging a little bit, I figure out that the parent id sent to the slug generation is itself. See here the code:

https://github.com/Sylius/Sylius/blob/1e492e77eb7b3aed83e8fa600e2a43f41439d6b2/src/Sylius/Bundle/AdminBundle/Resources/views/Taxon/_slugField.html.twig#L4

It does not make sense to me to generate slug like `category/category` while it's actually `category`.

Is it a bug or is there a real feature hiding there?

**Comment by Rafikooo** [comment]
This bug has been fixed in current versions of Sylius. The problematic template code has been removed, andvslug generation now correctly handles top-level taxons without parent references.

If you're experiencing similar issues with recent versions of Sylius, please open a new discussion with your Sylius version and reproduction steps.

Thank you for reporting this issue :)
